.werft/jobs/build/self-hosted-upgrade-tests.ts

```diff
@@ -114,7 +114,7 @@ export async function triggerSelfHostedPreview(werft: Werft, config: JobConfig,
 
     exec(`git config --global user.name "${username}"`);
 
-    annotation = `${annotation} -a channel=${replicatedChannel} -a preview=true -a skipTests=true -a deps=external`;
+    annotation = `${annotation} -a channel=${replicatedChannel} -a preview=true -a deps=external`;
```

Comment:
Nit: should we make the command explicit?
```suggestion
    annotation = `${annotation} -a channel=${replicatedChannel} -a preview=true -a runTests=false -a deps=external`;
```

Reply:
sure, this is the default behavior and hence I decided to omit it to reduce the line length. :see_no_evil: I am happy to add it. Committing

Reply:
Yeah, I know what you mean. I can see it both ways - I guess that because we're changing the behaviour, it could be useful making it explicit (but who looks through this code really?)